app/services/LobbyManagerService.js

```diff
@@ -7,6 +7,7 @@ function LobbyManagerService(){
   return {
     getLobbies: getLobbies,
     create: create,
+    getLobbyID: lobbyCount
```

Comment:
This is far from reliable way to define a lobby Id.
Also, following the set naming convention, this member should be called `getLobbyId`